ui_framework/src/components/call_out/_call_out.scss

```diff
@@ -0,0 +1,57 @@
+.kuiCallOut {
+  padding: $kuiSizeM $kuiSizeL;
+  border-left: 4px solid $kuiColorPrimary;
+  background-color: rgba($kuiColorPrimary, 0.1); /* 1 */
```

Comment:
I'd prefer if we avoid opacity in places where we can use flat values (tintOrShade works perfectly fine here). I know opacity is easy for theming, but I'd like to avoid the render problems that opacity can aggravate. Anything that we can do to help the slowdowns that K6 suffers from (think of the slowness around resizing the discover page) I think we should try to do. tintOrShade also lets us be more specific and apply separate values.

Opacity is great for animations and shadows. If it's just defining a color, lets use a color.